.github/workflows/ci.yml

```diff
@@ -42,7 +42,6 @@ jobs:
         uses: ocaml/setup-ocaml@v3
         with:
           ocaml-compiler: ${{ matrix.ocaml-compiler }}
-          dune-cache: true
```

Comment:
Sorry for the very late response! :bow: (I have been tied up with other tasks for a while.)

Let me check for sure: is the removal of `dune-cache: true` a workaround for Windows?

Reply:
Oops

Reply:
Fixed.

Reply:
It should be available on all operating systems btw!